dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/subcommands/commonsteps/Common.java

```diff
@@ -0,0 +1,44 @@
+package com.salesforce.dockerfileimageupdate.subcommands.commonsteps;
+
+import com.google.common.collect.*;
+import com.salesforce.dockerfileimageupdate.model.*;
+import com.salesforce.dockerfileimageupdate.process.*;
+import com.salesforce.dockerfileimageupdate.utils.*;
+import net.sourceforge.argparse4j.inf.*;
+import org.kohsuke.github.*;
+import org.slf4j.*;
+
+import java.io.*;
+import java.util.*;
+
+public class Common {
+    private static final Logger log = LoggerFactory.getLogger(Common.class);
+    public void prepareToCreatePullRequests(final Namespace ns,
```

Comment:
Let's move this to `util` and name the class `PullRequests` and the method would be `prepareToCreate`. 

The callers would look like: `new PullRequests().prepareToCreate(...)`

Note that if we were to do it all over again, we wouldn't name anything `util`, but that is something we'd fix in a later refactoring.